src/main/kotlin/no/nav/familie/ba/sak/kjerne/brev/BrevUtil.kt

```diff
@@ -271,3 +307,96 @@ fun hentVirkningstidspunktForDødsfallbrev(
+        .values.kombiner().perioder().splitPerTidsenhet(LocalDate.now().tilMånedTidspunkt())
+        .groupBy { utbetalingEøsPeriode -> utbetalingEøsPeriode.fraOgMed.tilYearMonth().tilMånedÅrMedium() }
+        .mapValues { (_, utbetalingEøsPerioder) ->
+            utbetalingEøsPerioder.mapNotNull { it.innhold }.flatten()
+        }
+        .mapValues { (_, utbetalingerEøs) ->
+            hentUtbetalingMndEøs(utbetalingerEøs = utbetalingerEøs)
+        }
+}
+
+private fun hentUtbetalingEøs(
+    aktørOgYtelseType: Pair<Aktør, YtelseType>,
+    andelUpbOgValutakurs: AndelUpbOgValutakurs,
+): UtbetalingEøs {
+    return UtbetalingEøs(
+        fnr = aktørOgYtelseType.first.aktivFødselsnummer(),
+        ytelseType = aktørOgYtelseType.second,
+        satsINorge = andelUpbOgValutakurs.andelForVedtaksperiode.sats,
+        utbetaltFraAnnetLand =
+            when (andelUpbOgValutakurs.utenlandskPeriodebeløp) {
+                null -> null
+                else ->
+                    try {
+                        UtbetaltFraAnnetLand(
+                            beløp = andelUpbOgValutakurs.utenlandskPeriodebeløp.tilKalkulertMånedligBeløp()!!.toBigInteger().intValueExact(),
+                            valutakode = andelUpbOgValutakurs.utenlandskPeriodebeløp.valutakode!!,
+                            beløpINok = (andelUpbOgValutakurs.utenlandskPeriodebeløp.tilMånedligValutabeløp()!! * andelUpbOgValutakurs.valutakurs.tilKronerPerValutaenhet())!!.toBigInteger().intValueExact(),
+                        )
+                    } catch (exception: NullPointerException) {
+                        throw Feil("Kan ikke opprette UtbetaltFraAnnetLand for periode med utenlandsk periodebeløp da ett eller flere av de påkrevde feltene er null: kalkulertMånedligBeløp = ${andelUpbOgValutakurs.utenlandskPeriodebeløp.tilKalkulertMånedligBeløp()}, valutkode = ${andelUpbOgValutakurs.utenlandskPeriodebeløp.valutakode} valutakurs = ${andelUpbOgValutakurs.valutakurs.tilKronerPerValutaenhet()}")
+                    }
+            },
+        utbetaltFraNorge = andelUpbOgValutakurs.andelForVedtaksperiode.kalkulertUtbetalingsbeløp,
+    )
+}
+
+private fun hentUtbetalingMndEøs(utbetalingerEøs: List<UtbetalingEøs>): UtbetalingMndEøs {
+    val summertUtbetaltFraAnnetLand = utbetalingerEøs.sumOf { utbetalingEøs -> utbetalingEøs.utbetaltFraAnnetLand?.beløpINok ?: 0 }
+    return UtbetalingMndEøs(
+        utbetalinger = utbetalingerEøs,
+        oppsummering =
+            UtbetalingMndEøsOppsummering(
+                summertSatsINorge = utbetalingerEøs.sumOf { utbetalingEøs -> utbetalingEøs.satsINorge },
+                summertUtbetaltFraAnnetLand =
+                    when (summertUtbetaltFraAnnetLand) {
+                        0 -> null
+                        else -> summertUtbetaltFraAnnetLand
+                    },
+                summertUtbetaltFraNorge = utbetalingerEøs.sumOf { utbetalingEøs -> utbetalingEøs.utbetaltFraNorge },
```

Comment:
Trenger vi egentlig å navngi input her?
```suggestion
                summertUtbetaltFraNorge = utbetalingerEøs.sumOf { it.utbetaltFraNorge },
```